### ISSUE TITLE
False positive / VIP.ValidatedSanitizedInput

### ISSUE THREAD

**Issue by jrfnl** [WordPress-Coding-Standards/WordPress-Coding-Standards#359]
```php
if ( ! isset( $_POST['shortcode'] ) || $_POST['shortcode'] === '' ) {
	exit( '-1' );
}
```

IMHO the above usage should be ok. Yes, the value is not sanitized, but this comparison does not yield a situation where the non-sanitized variable could in any way be dangerous.

`Detected usage of a non-sanitized input variable: $_POST (WordPress.VIP.ValidatedSanitizedInput.InputNotSanitized)`

**Comment by JDGrimes** [comment]
Duplicate of #186.

**Comment by jrfnl** [comment]
Good catch, thanks!
